cypress/integration/core-opensearch-dashboards/opensearch-dashboards/apps/query_enhancements/saved_search.spec.js

```diff
@@ -0,0 +1,513 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  END_TIME,
+  START_TIME,
+  INDEX_PATTERN_WITH_TIME,
+  INDEX_WITH_TIME_1,
+  INDEX_WITH_TIME_2,
+} from './constants';
+import { SECONDARY_ENGINE } from '../../../../../utils/constants';
+import { v4 as uuid } from 'uuid';
+
+const workspaceName = uuid();
+// datasource name must be 32 char or less
+const datasourceName = uuid().substring(0, 32);
+
+const allowedSearchOperations = {
```

Comment:
I think this might be used by other tests. Should we move it to constant?

Reply:
yes can do, i will also add a comment on it to make it more helpful to other devs

Reply:
moved it to constant